_data-prepper/common-use-cases/sampling.md

```diff
@@ -0,0 +1,78 @@
+
+In this example, only 100 events per second with a status code of `200` are sent to the sink from a given IP address. It drops all excess events from the configured time window.
+
+```json
+...
+  processor:
+   - aggregate:                                                                                                                                          
+        identification_keys: ["clientip"]                                                                                                      
+        action:                                                                                                                                           
+          rate_limiter:                                                                                                                                   
+            events_per_second: 100                                                                                                                        
+            when_exceeds: drop
+        when: "/status == 200"  
+...
+```
+
+If you instead set the `when_exceeds` option to `block`, the processor will block the pipeline until the time window completes. Then it will process the blocked events.
+
+## Percentage sampling
+
+Use the `percent_sampler` action within the `aggregate` processor to limit the number of events that are sent to a sink. All excess events will be dropped.
+
+In the following example, only 20 percent of events with a status code of `200` are sent to the sink from a given IP address:
+
+```json
+...
+  processor:
+  - aggregate:                                                                                                                                          
+        identification_keys: ["clientip"]  
+        duration :                                                                                                    
+        action:                                                                                                                                           
+          percent_sampler:                                                                                                                                   
+            percent: 20                                                                                                                        
+        when: "/status == 200" 
+...
+```
+
+## Tail sampling
+
+Use the `tail_sampler` action within the `aggregate` processor to sample events based on a set of defined policies. This action waits for an aggregation to complete across different aggregation periods based on the configured wait period. When an aggregation is complete, and if it matches the specific error condition, it's sent to the sink. Otherwise, only a configured percentage of events is sent to the sink.
```

Comment:
```suggestion
Use the `tail_sampler` action within the `aggregate` processor to sample events based on a set of defined policies. This action waits for an aggregation to complete across different aggregation periods based on the configured wait period. When an aggregation is complete, and if it matches the specific error condition, it is sent to the sink. Otherwise, only a configured percentage of events is sent to the sink.
```